nvtabular/torch_dataloader.py

```diff
@@ -60,23 +59,23 @@ def __len__(self):
 
     def __iter__(self):
         for idx in range(0, self.num_samples, self.batch_size):
-            tens = [tensor[idx : idx + self.batch_size] for tensor in self.tensors]
-            yield tens[0], tens[1], tens[2]
-            del tens
+            yield [
+                tensor[idx : idx + self.batch_size] if tensor is not None else None
+                for tensor in self.tensors
+            ]
 
     def shuffle(self):
         idx = torch.randperm(self.num_samples, dtype=torch.int64)
         self.tensors = [tensor[idx] for tensor in self.tensors]
 
 
-def _to_tensor(gdf: cudf.DataFrame, dtype, to_cpu=False):
+def _to_tensor(gdf: cudf.DataFrame, dtype):
     if gdf.empty:
         return
     g = gdf.to_dlpack()
-    cols = gdf.columns
     t = from_dlpack(g).type(dtype)
     del g, gdf
-    return t, cols
```

Comment:
we going to want these columns later... in order to create the columns / tensors class (aka dictionary)... wont we?

Reply:
We weren't using it: the only time we called _to_tensor we didn't use that return value (and went  ```return [cats[0], conts[0], label[0]]``` to only access the tensor and not the columns). Its simpler to just not include it here

Reply:
I think we will have to have a substantial rewrite before using a columns/tensor class - and we're not using this info now, and it's simpler just to leave off in the meantime